src/libertem/api.py

```diff
@@ -691,9 +705,51 @@ def _create_local_executor(self):
         cores = psutil.cpu_count(logical=False)
         if cores is None:
             cores = 2
-        return DaskJobExecutor.make_local(
-            cluster_kwargs={"threads_per_worker": 1, "n_workers": cores},
-            client_kwargs={'set_as_default': False}
+        workers_spec = {}
+
+        cpu_base_spec = {
+            "cls": dd.Nanny,
+            "options": {"nthreads": 1, "resources": {"CPU": 1, "compute": 1}}
+        }
+
+        # Service workers not for computation
```

Comment:
Hmm, maybe we can make the service workers multi-threaded? They will mostly block on I/O anyways, so we probably won't have GIL problems. Just need to take care w/ thread safety.

Should the service worker define a `{"service": 1}` resource, which then can be acquired by `executor.run_function(service=True)` and similar?

Reply:
I could make the service worker multithreaded, just a question of setting options in the `cluster_spec()` function. Probably we should leave that until we have this covered in benchmarks. That way we can see if it makes any difference.

Are there use cases where restricting service tasks to specific workers would be better than just running on any worker?

Reply:
> Are there use cases where restricting service tasks to specific workers would be better than just running on any worker?

I don't think there are any use cases yet, let's keep it simple for now.